web/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,56 @@
+require 'user_pb'
+
+class UsersController < ApplicationController
+  before_action :assign_user, only: %i[show omniauth]
+
+  def new
+    @form = CreateUserForm.new
+  end
+
+  def create
+    @form = CreateUserForm.new(params[:user])
+    if @form.valid?
+      req = Soapbox::CreateUserRequest.new(name: @form.name, email: @form.email, password: @form.password)
+      @user = $api_user_client.create_user(req)
+      session[:current_user_email] = @user.email
+      redirect_to user_path(@user)
```

Comment:
Ahh, this is a routes issue. Added a line to `routes.rb` that makes the URL for the show page `/user/profile` instead.